web/components/view/ShowHideTextView/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import arrow from '../../../assets/arrow.svg';
+
+class ShowHideTextView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {isToggleOn: true};
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(state => ({
+      isToggleOn: !state.isToggleOn
+    }));
+  }
+
+  render() {
+    return (
+      <div className="show-hide card secondary-outline">
+        <div className="pad-sides-s pad-ends-s user-select-none clickable flex jc-between" onClick={this.handleClick}>
+          <label className="show-hide-outer-text margin-ends-s margin-right-s clickable">{this.props.outerText}</label>
```

Comment:
I think `outerText` isn't that obvious? why not just name it `label` since it's a label?